README.adoc

```diff
@@ -309,24 +400,94 @@ The default host name for Docker Desktop is `localhost`.
 The default host name for minikube is {minikube-ip}. Otherwise it can be found using the `minikube ip` command.
 --
 
-Then `curl` or visit the following URLs to access your microservices, substituting the appropriate host name:
+Then use `curl` or visit the following URLs to access your microservices, substituting the appropriate host name:
 
-- `{system-api}`
-- `{inventory-api}/system-service`
+* `{system-api}`
+* `{inventory-api}/system-service`
 
 The first URL returns system properties and the name of the pod in an HTTP header called `X-Pod-Name`.
 To view the header, you may use the `-I` option in the `curl` when making a request to `{system-api}`.
-The second URL adds properties from `system-service` to the inventory
-{kube} Service. Visiting `{inventory-api}/[kube-service]` in general adds to the inventory
-depending on whether `kube-service` is a valid {kube} Service that can be accessed.
+The second URL adds properties from `system-service` to the inventory {kube} Service. 
+Visiting `{inventory-api}/[kube-service]` in general adds to the inventory
+depending on whether `kube-service` is a valid {kube} service that can be accessed.
+endif::[]
+
+// Cloud-hosted guide instruction
+ifdef::cloud-hosted[]
+Run the following command to get the node IP for the `system` service:
+
+```
+kubectl describe pod system | grep Node
+```
+{: codeblock}
+
+The output will show the node IP that will be used to access the service later. 
+It will appear in a format similar to the following:
+
+```
+Node:         10.114.85.140/10.114.85.140
+Node-Selectors:  <none>
+```
+
+Here the IP is `10.114.85.140` for the system service.
```

Comment:
Here the IP is `10.114.85.140` for the system service.
--->
The IP for the system service is `10.114.85.140`.